netlogo-gui/src/main/ide/CodeCompletionPopup.scala

```diff
@@ -38,6 +38,20 @@ case class CodeCompletionPopup() {
       suggestionDisplaylist.setLayoutOrientation(JList.VERTICAL)
       suggestionDisplaylist.setVisibleRowCount(10)
       suggestionDisplaylist.setCellRenderer(new SuggestionListRenderer(eA))
+      
+      suggestionDisplaylist.addMouseListener(new MouseListener {
```

Comment:
You can use `MouseAdapter` here so you don't have to specify all the empty methods